src/pages/journal/journal-components/journal-candidate/journal-candidate.scss

```diff
@@ -0,0 +1,32 @@
+journal-candidate {
+    & > div {
+      height: 100%;
+      padding: 0.3em 0.75em;
+      background-color: map-get($colors-gds, gds-grey-4);
+      border-radius: 13px;
+      @include box-shadow(0px, 2px, 4px, rgba(0, 0, 0, 0.55));
+      div {
+        height: 50%;
+      }
+      &.journal-candidate-test-complete-text {
+        color: map-get($colors-gds, gds-grey-1);
+      }
+      img {
+        width: 2.25em;
```

Comment:
The category image is quite small in portrait mode.

Reply:
Notice that, it's using an em - we should be using px right?

Reply:
Think we're generally preferring `em` except for things like border(-radius) - we don't have a definitive list.

Reply:
Fixed issue with small images. Updated with px values from abstract design.